### PR TITLE
[Docs] Relationships: Grammar fix in the "Design Relationships and Restrictions" section

### LINKED ISSUE #9997: [Docs Designs] Misplaced full stop
### Current State
Link: [Meshery Docs](https://docs.meshery.io/concepts/logical/designs)

Section: Design Relationships and Restrictions

Currently the sentence "Designs can be converted into reusable Patterns. Creating a Pattern. involves replacing the values of the variables in the design with the values provided by another user. " has a misplaced full stop.

### Desired State

Remove the full stop so the sentence reads as follows: "Designs can be converted into reusable Patterns. Creating a Pattern involves replacing the values of the variables in the design with the values provided by another user. "

#### Screenshot

Screenshot 2024-01-17 201056

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by neerajmanivarnan**
**Notes for Reviewers**
This PR fixes the misplaced full stop issue referred by #9997 

This PR fixes #9997

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 86ecf9a58baf245b726db4a842ed208526d16145 at: https://65a82e4649068a008769b8fc--meshery-docs-preview.netlify.app

File: docs/pages/guides/configuration-management/importing-applications.md [hunk 1]
```diff
@@ -50,7 +52,7 @@ This command enable users to import their existing applications from sources as
 
 **Step 1: Access the Meshery UI**
 
-To import a Docker Compose app into Meshery using the Meshery UI, you must first [install Meshery](../installation/quick-start.md)
+To import a Docker Compose app into Meshery using the Meshery UI, you must first [install Meshery](/installation/quick-start)
```
Comment (sandramsc) [inline docs/pages/guides/configuration-management/importing-applications.md:55]: Why are there changes here, this isn't part of the issue description.

**Comment by sandramsc** [comment]
Please consider updating the heading of the PR to be more closely worded with the related issue, it makes for a more efficient review for contributors and maintainers.

i.e Repeating the issue title --> [Fix Docs Designs] Misplaced full stop

File: docs/pages/guides/configuration-management/importing-applications.md (hunk 1 above)
Comment (neerajmanivarnan) [reply docs/pages/guides/configuration-management/importing-applications.md:55]: Those are commits from the issue #9991

**Comment by leecalcote** [comment]
@neerajmanivarnan send word when you've incorporated @sandramsc's feedback>

File: docs/pages/guides/configuration-management/importing-applications.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/guides/configuration-management/importing-applications.md:54]: Thanks for this review, @sandramsc 👏

**Comment by leecalcote** [comment]
> @neerajmanivarnan send word when you've incorporated @sandramsc's feedback>

@neerajmanivarnan here PR renaming feedback takes all of 4 seconds to update...

File: docs/pages/guides/configuration-management/importing-applications.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ permalink: guides/configuration-management/importing-apps
 category: configuration
 type: guides
 language: en
+redirect_from:
+  - /installation/quick-start.md
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/importing-applications.md:10]: No. This is not only inaccurate, but improper syntax.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/guides/configuration-management/importing-applications.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/guides/configuration-management/importing-applications.md:54]: @neerajmanivarnan, don't include commits from a separate PR into another PR.

File: docs/pages/guides/configuration-management/importing-applications.md (hunk 2 above)
Comment (leecalcote) [inline docs/pages/guides/configuration-management/importing-applications.md:10]:
```suggestion
```

**Review by leecalcote — APPROVED** [review]
Redirect_from removed.

**Comment by leecalcote** [comment]
Thanks for contributing, @neerajmanivarnan 👍

**Comment by leecalcote** [comment]
Whoo-hoo! A first PR!! Congrats, @neerajmanivarnan 🎈

**Pull request merged**